### PR TITLE
docs: translate `renderToString` section

### LINKED ISSUE #462: `renderToString`
Page: https://id.react.dev/reference/react-dom/server/renderToString

Assignee: @hisamafahri

### PR DESCRIPTION
**Pull request by hisamafahri**
Close #462

### CODE REVIEW

**Review by r17x — CHANGES_REQUESTED** [review]
Please use *asterisk* (*) instead of *underscore* (_)

File: src/content/reference/react-dom/server/renderToString.md
Comment (r17x) [inline src/content/reference/react-dom/server/renderToString.md:111]:
```suggestion
Mengimpor `react-dom/server` **pada klien** meningkatkan ukuran bundel Anda secara tidak perlu dan harus dihindari. Jika Anda perlu me-*render* beberapa komponen ke HTML di peramban, gunakan [`createRoot`](/reference/react-dom/client/createRoot) dan baca HTML dari DOM:
```

**Review by r17x — CHANGES_REQUESTED** [review]

**Comment by mhaidarhanif** [comment]
@hisamafahri Each *tree* need to be translated to `pohon`

Need help with others to make sure we translate `*tree*` to `pohon (*tree*)` as in all other translated pages / merged PRs. @r17x @resir014 @zainfathoni @mazipan

**Comment by hisamafahri** [comment]
> Each `tree` need to be translated to `pohon`
> 
> Need help with others to make sure we translate `tree` to `pohon (*tree*)` as in all other translated pages / merged PRs @r17x @resir014 @zainfathoni @mazipan


@mhaidarhanif I've thought about that too. But looking back using `pohon` is really confusing from the reader's standpoint.

I've asked few colleagues of mine, and they prefer to use `tree` instead.

But, if the maintainers have decided that, I'll be happy to follow that 😁

**Comment by zainfathoni** [comment]
> > Each `tree` need to be translated to `pohon`
> 
> > 
> 
> > Need help with others to make sure we translate `tree` to `pohon (*tree*)` as in all other translated pages / merged PRs @r17x @resir014 @zainfathoni @mazipan
> 
> 
> 
> 
> 
> @mhaidarhanif I've thought about that too. But looking back using `pohon` is really confusing from the reader's standpoint.
> 
> 
> 
> I've asked few colleagues of mine, and they prefer to use `tree` instead.
> 
> 
> 
> But, if the maintainers have decided that, I'll be happy to follow that 😁

It's just a matter of habit. I have been using `pohon`, `cabang`, `akar`, and `daun` for years since my college and all the students in the whole class had no problems with it. We understood the modules well.

And you only need to do `pohon (*tree*)` once in the first usage of the page. After that, you mau omit `(*tree*)` and only use `pohon` in it.

**Comment by hisamafahri** [comment]
@zainfathoni Noted. Looks like we need to get used to use terms in Bahasa Indonesia more often from now on. 😂

File: src/content/reference/react-dom/server/renderToString.md
Comment (zainfathoni) [inline src/content/reference/react-dom/server/renderToString.md:7]:
```suggestion
`renderToString` tidak mendukung *streaming* atau menunggu data. [Lihat beberapa alternatifnya.](#alternatives)
```

File: src/content/reference/react-dom/server/renderToString.md
Comment (zainfathoni) [inline src/content/reference/react-dom/server/renderToString.md:135]:
```suggestion
Jika beberapa komponen ditangguhkan (misalnya, karena ditentukan dengan [`lazy`](/reference/react/lazy) atau mengambil data), `renderToString` tidak akan menunggu kontennya diselesaikan. Sebagai gantinya, `renderToString` akan menemukan batas [`<Suspense>`](/reference/react/Suspense) terdekat di atasnya dan merender prop `fallback` di HTML. Konten tidak akan muncul hingga kode klien dimuat.

```

Please restore the empty lines. The empty lines are necessary to split paragraphs in Markdown.

**Review by zainfathoni — CHANGES_REQUESTED** [review]
I have some minor suggestions so far from skimming the text. I'll review it more thoroughly later tonight.

File: src/content/reference/react-dom/server/renderToString.md
Comment (hisamafahri) [reply src/content/reference/react-dom/server/renderToString.md:135]: I am not really sure why GitHub display it as a deleted line. But I don't delete it 😄 👍 
image

Comment (zainfathoni) [reply src/content/reference/react-dom/server/renderToString.md:135]: Oops, my bad. Thanks for clarifying. 😅

File: src/content/reference/react-dom/server/renderToString.md
Comment (zainfathoni) [reply src/content/reference/react-dom/server/renderToString.md]: Please don't forget to italicize the `streaming` word, @hisamafahri. Thanks!

Comment (hisamafahri) [reply src/content/reference/react-dom/server/renderToString.md]: Ah, missed that one. Should be fixed now

**Review by r17x — APPROVED** [review]

File: src/content/reference/react-dom/server/renderToString.md
Comment (mazipan) [inline src/content/reference/react-dom/server/renderToString.md:53]: Typo
```suggestion
* `renderToString` bekerja di peramban, tapi menggunakannya di kode klien [tidak direkomendasikan.](#removing-rendertostring-from-the-client-code)
```

File: src/content/reference/react-dom/server/renderToString.md
Comment (mazipan) [inline src/content/reference/react-dom/server/renderToString.md:61]:
```suggestion
Panggil `renderToString` untuk me-*render* aplikasi Anda ke *string* HTML yang dapat Anda kirim dengan respons server Anda:
```

File: src/content/reference/react-dom/server/renderToString.md
Comment (mazipan) [inline src/content/reference/react-dom/server/renderToString.md:66]:
```suggestion
// Sintaksis pengendali rute bergantung pada framework backend Anda
```

File: src/content/reference/react-dom/server/renderToString.md
Comment (mazipan) [inline src/content/reference/react-dom/server/renderToString.md:88]: Not sure about this.

```suggestion
`renderToString` mengembalikan *string* dengan seketika, sehingga tidak mendukung *streaming* atau menunggu data.
```

File: src/content/reference/react-dom/server/renderToString.md
Comment (mazipan) [inline src/content/reference/react-dom/server/renderToString.md:90]:
```suggestion
Saat memungkinkan, kami merekomendasikan untuk menggunakan alternatif yang berfitur lengkap ini:
```

File: src/content/reference/react-dom/server/renderToString.md
Comment (mazipan) [inline src/content/reference/react-dom/server/renderToString.md:125]:
```suggestion
Memanggil [`flushSync`](/reference/react-dom/flushSync) diperlukan agar DOM diperbarui sebelum membaca [`innerHTML`](https://developer.mozilla.org/en-US/docs/Web/API/Elemen/innerHTML).
```

File: src/content/reference/react-dom/server/renderToString.md
Comment (hisamafahri) [reply src/content/reference/react-dom/server/renderToString.md]: Looks good, I'll take it 👍

**Review by mazipan — APPROVED** [review]

**Comment by r17x** [comment]
Need review based on request changes by @zainfathoni

**Comment by mhaidarhanif** [comment]
@zainfathoni can check and approve if okay.

**Pull request merged**
